test/solcjsTests.sh

```diff
@@ -53,11 +53,6 @@ DIR=$(mktemp -d)
     rm -f soljson.js
     cp "$SOLJSON" soljson.js
 
-    # ensure to use always 0.5.0 sources
```

Comment:
The proper version switch should now be done here: https://github.com/ethereum/solc-js/pull/383.